app/scripts/lib/transaction/util.ts

```diff
@@ -156,7 +157,7 @@ export async function addTransaction(
             data: transactionParams.data,
           },
         ],
-      };
+      });
 
       const securityAlertId = uuid();
 
```

Comment:
There's a considerable amount of duplication below with the content of `ppom-middleware.ts`, so I'd recommend generalising the logic into a method in `ppom-util.ts` such as `validateWithPPOM`.

I didn't want to touch it in this PR in order to avoid introducing additional risk.